cmd/teststand/processor/teststand.go

```diff
@@ -0,0 +1,97 @@
+package processor
+
+import (
+	"fmt"
+	percentileCalc "github.com/orktes/go-percentile"
```

Comment:
I'm not 100% sure about this dep. It has 0 stars, and the last commit was 6 years ago. 

There is a possibility that we will need to calc percentiles as a window function (e.g. for every minute). Premature optimization is evil, but in this case it is better to use our own MVP package to calculate percentiles (implement functionality as needed). Or maybe we need to take another look at this and find another dependency that is more popular and flexible.

Also you can think not only about percentiles. Imagine an aggregator that collects all the information about run execution, calculates arbitrary metrics and returns which are needed. You can keep the logic of result interpretation in this aggregator.

Reply:
I guess, custom calculator will be good enough. Let's make our own stuff.